src/brokers/consorsbank.js

```diff
@@ -57,6 +57,13 @@ const findDateBuySell = content => {
   if (lineNumber <= 0) {
     return undefined;
   }
+  // Might be a super old file, test from 2003
+  if (
+    content[lineNumber + 1].toLowerCase().startsWith('kauf') &&
```

Comment:
So to be prepared for old sell documents, we can check here for:
```suggestion
    (content[lineNumber + 1].toLowerCase().startsWith('kauf') || content[lineNumber + 1].toLowerCase().startsWith('verkauf')) &&
```